### DISCUSSION TITLE
Missing Options in Exposure time finder

### DISCUSSION THREAD

**Discussion by WallyW0** [sn4k3/UVtools#819] [Q&A]
### System

```Text
UVtools v4.0.6 X64
Operative system: Microsoft Windows 10.0.22621 X64
Processor: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
Processor cores: 8
Memory RAM: 7.24 / 7.89 GB
Runtime: win10-x64
Framework: .NET 6.0.25
AvaloniaUI: 11.0.6
OpenCV: 4.8.1

Path:       C:\Program Files\UVtools\
Executable: C:\Program Files\UVtools\UVtools.exe
Loaded file: _Rook_SATURN_04h00m_54ml_copy.ctb [Version: 2] [Class: ChituboxFile]
```

### Printer and Slicer

- Printer: Elegoo Saturn
- Slicer: Chitubox / Lychee

### Description of the bug

The problem is that some options are not displayed when using the Exposure time finder tool. 

I refer to the tutorial video "Find the Perfect Exposure Time for Resin 3D Printing With UVTools!" where all possible options are shown.

The following options are missing in my version:

-Multiple Layer high
-Multiple Exposures

Screenshot:

![2024-01-07_10h16_12](https://github.com/sn4k3/UVtools/assets/155881016/39952a33-2ec9-4193-a862-b35df85c7409)

### How to reproduce

1. Load File
2. Go to Calibration Exposure time finder
3. Scroll down to the available options
4. Only Multiple Brightness is available

### Files

[Rook_SATURN.zip](https://github.com/sn4k3/UVtools/files/13852849/Rook_SATURN.zip)

**Comment by sn4k3** [comment]
They are not missing, your problem is this: `Loaded file: _Rook_SATURN_04h00m_54ml_copy.ctb [Version: 2] [Class: ChituboxFile]`
Version 2 of CTB don't allow per layer settings, you need a file with version 3 or 4.

See: https://github.com/sn4k3/UVtools/discussions/806
